source/reverse-sync.txt

```diff
@@ -0,0 +1,189 @@
+        that write operations are stopped for the collections included
+        by the filter.
+
+   .. step:: Send a :ref:`commit <c2c-api-commit>` request to ``mongosync``.
+
+      If you started multiple ``mongosync`` instances for your
+      migration, you must issue a commit request for each ``mongosync``
+      instance.
+
+      Request
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/commit.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/success.json
+         :language: json
+         :copyable: false
+
+      .. note::
+
+         After you submit a ``commit`` request, call the ``progress`` endpoint
+         to ensure that the ``mongosync`` state is ``COMMITTING`` or
+         ``COMMITTED``.
+
+      Once you complete this step, ``mongosync`` blocks writes on the source cluster.
+
+   .. step:: Wait until you can perform writes on the destination cluster.
+
+      Call the ``progress`` endpoint to determine if ``canWrite`` is
+      ``true``. If ``canWrite`` is ``false``, wait until ``progress``
+      shows ``canWrite`` is ``true``.
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: bash
+
+            curl -sS localhost:27182/api/v1/progress -XGET | jq ".progress.canWrite"
+
+         .. output::
+            :language: json
+
+            true
+
+      When you finish reversing the direction of your sync operation,
+      ``mongosync`` replicates any writes that you perform on the destination 
+      cluster after this step to the source cluster.
+
+   .. step:: Verify data transfer.
+
+      Verify the successful sync of data from the source to the
+      destination cluster.
+
+      For more information, see :ref:`c2c-verification`.
+
+   .. step:: Enable application writes on the destination cluster.
+
+      To enable writes, update :dbcommand:`setUserWriteBlockMode`:
+
+      .. code-block:: javascript
+
+         db.adminCommand(
+            {
+               setUserWriteBlockMode: 1,
+               global: false
+            }
+         )
+
+      Then, transfer your application workload to the destination cluster.
+
+   .. step:: Call the ``progress`` endpoint to determine the status of the ``mongosync`` process.
+
+      When the ``mongosync`` progress response indicates that the
+      ``mongosync`` state is ``COMMITTED``, the cutover process is
+      complete.
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: bash
+
+            curl -sS localhost:27182/api/v1/progress -XGET | jq ".progress.state"
+
+         .. output::
+            :language: json
+
+            "COMMITTED"
+
+   .. step:: Call the ``reverse`` endpoint.
+   
+      Call the ``reverse`` endpoint to reverse the direction of your sync 
+      operation. Your original source cluster becomes your new destination cluster 
+      and vice versa. 
```

Comment:
[suggestion]

Consider rewording this. I don't think we want to use "vice versa" or other latin terms in the docs unless totally necessary.